RotatedArray/src/ua/goit/alg/RotatedArray.java

```diff
@@ -0,0 +1,28 @@
+package ua.goit.alg;
+
+public class RotatedArray {
+
+    public static int searchInRotatedArray(int array[], int length, int value) {
+	int left = 0;
+	int right = length - 1;
+
+	while (left < right) {
+	    int middle = left + right / 2;
+	    if (array[middle] == value) 
+		return middle;
```

Comment:
we need an empty line after this. also it's recommended to use curly braces here